pkg/injector/sentry/sentry.go

```diff
@@ -109,7 +111,7 @@ func (r *Requester) RequestCertificateFromSentry(ctx context.Context, namespace
 			Token:          token,
 			Namespace:      namespace,
 			TokenValidator: tokenValidator,
-		}, grpcRetry.WithMax(3), grpcRetry.WithPerRetryTimeout(time.Second*3))
+		}, grpcRetry.WithMax(10), grpcRetry.WithPerRetryTimeout(time.Second*3))
```

Comment:
Then do we need to set a "WithMax"? Could we just abort on when the context was canceled, which happens automatically?

```suggestion
		}, grpcRetry.WithPerRetryTimeout(2*time.Second))
```

Reply:
WithMax is required to enable retries. Only having `WithPerRetryTimeout` will not enable it. Context cancellation will not retry the request, it will serve as a 30s timeout for a single request.